source/common/stats/thread_local_store.cc

```diff
@@ -75,6 +94,30 @@ void ThreadLocalStoreImpl::shutdownThreading() {
   shutting_down_ = true;
 }
 
+void ThreadLocalStoreImpl::mergeHistograms(PostMergeCb merge_complete_cb) {
+  if (!shutting_down_) {
+    tls_->runOnAllThreadsWithBarrier(
+        [this]() -> void {
+          for (ScopeImpl* scope : scopes_) {
+            for (auto histogram : tls_->getTyped<TlsCache>().scope_cache_[scope].histograms_) {
```

Comment:
explicit type for histogram, since we have multiple flavors.

Reply:
i have added the explicit type in the next assignment.